packages/flutter_tools/lib/src/windows/visual_studio.dart

```diff
@@ -190,11 +190,16 @@ class VisualStudio {
   }
 
   /// Checks if the given installation has issues that the user must resolve.
+  /// 
+  /// Returns false if the required information is missing since older versions
+  /// of Windows might not include them.
   bool installationHasIssues(Map<String, dynamic>installationDetails) {
     assert(installationDetails != null);
-    assert(installationDetails[_isCompleteKey] != null);
-    assert(installationDetails[_isRebootRequiredKey] != null);
-    assert(installationDetails[_isLaunchableKey] != null);
+    if (installationDetails[_isCompleteKey] == null ||
+        installationDetails[_isRebootRequiredKey] == null ||
+        installationDetails[_isLaunchableKey] == null) {
```

Comment:
I'd rather we not treat these as a monolithic block, where we don't check them at all if any one is missing. Instead this can be:

```
if (installationDetails[_isCompleteKey] != null && !installationDetails[_isCompleteKey]) {
  return true;
}
[similarly for the others]
return false;
```
if (

Reply:
Done